samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+	<!-- 
+		Microsoft ResX Schema
+
+		Version 1.3
```

Comment:
This doesn't look correct for the RESX tool nF uses.
Make sure to set the tool the nF resource parser.

Reply:
🤔 perhaps this is to do with the fact I created the sample when VS2019 support was not available... I will check...

Reply:
There might be a problem with the VS2019 extension adding nf resources... I am unable to do it! however the one already included seems to work fine!!!

Reply:
I have updated the file manually to be correct!